### PR TITLE
[Example] Basic stable diffusion demo

### PR DESCRIPTION
**Pull request by Aalanli**
A basic stable diffusion ui with gradio, however, eager and compiled modes have the same timing on my machine.

### CODE REVIEW

**Comment by Aalanli** [comment]
With the default settings.

**Comment by yaoyaoding** [comment]
Hi @Aalanli, thanks for the PR!

Let's wait for the support for the scaled_dot_product_attention. 

I have run this demo locally, and we will have out of memory issue (on my 3090) when we tried to tune in space 1 or 2. The reason might be the same as https://github.com/hidet-org/hidet/issues/217. We can merge this PR first (after we have scaled_dot_product), and then consider construct the model like https://github.com/hidet-org/hidet/blob/main/python/hidet/testing/models/resnet.py (use the weights from diffusers).

File: examples/stable-diffusion/main.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    else:
+        time_text += f"eager time: {eager_time}"
+    if compile_time == 0:
+        time_text += " | compile time: ..."
+    else:
+        time_text += f" | compile time: {compile_time}"
+    
+    return images, time_text
+
+def adjust_compile_opts(search_space=0, fp16=True, tensor_core=True, fp_16_red=False, cuda_graph=True, attn=False):
```
Comment (xinli-git) [inline examples/stable-diffusion/main.py:56]: attn is not used here? Should we give it a shot and see if the pictures are OK?

Comment (Aalanli) [reply examples/stable-diffusion/main.py:56]: Yes, I fixed a minor issue, but the pictures should be alright, at least with the default settings.

Comment (xinli-centml) [reply examples/stable-diffusion/main.py:56]: Thanks!

**Pull request closed without merging**